src/api/Live25API.js

```diff
@@ -0,0 +1,19 @@
+import masontodayAPI from './masontodayAPI';
+
+class Live25API {
+    constructor() {
+        this.urlprefix = 'api/25live';
+    }
+
+    async getData() {
```

Comment:
`getData` is not very descriptive -- what is the data that's being requested?

Reply:
Good point! Resolved with 82d55bb6bd8b6311bb6f2b9ac73a5b18df423827